app/views/errors/not_found.html.erb

```diff
@@ -3,17 +3,17 @@
 <div class="container text-center">
   <div class="row row-errorpage">
     <div class="col-xs col-sm col-md col-lg">
-      <div class="error-code bold pt-5">404</div>
+      <div class="error-code bold pt-5"><%= t("errors.not_found.error_code") %></div>
```

Comment:
Error code is not to be translated, we can just keep using [Arabic numerals](https://en.wikipedia.org/wiki/Arabic_numerals)